lib/passageidentity/auth.rb

```diff
@@ -1,41 +1,53 @@
-require "openssl"
-require "base64"
-require "jwt"
-require_relative "client"
-require_relative "../openapi_client"
+# frozen_string_literal: true
+
+require 'openssl'
+require 'base64'
+require 'jwt'
+require_relative 'client'
+require_relative '../openapi_client'
 
 module Passage
+  # The Passage::Auth class provides methods for authenticating requests and tokens
+  # using the Passage authentication service. It supports fetching and caching
+  # JSON Web Keys (JWKS) for token verification, and provides methods for
+  # validating JWTs and revoking user refresh tokens.
   class Auth
-    @@app_cache = {}
     def initialize(app_id, auth_strategy)
+      @app_cache = {}
       @app_id = app_id
       @auth_strategy = auth_strategy
 
       fetch_jwks
     end
 
-    def fetch_app()
-      begin
-        client = OpenapiClient::AppsApi.new
-        response = client.get_app(@app_id)
-        
-        return response.app
-      rescue Faraday::Error => e
-        raise PassageError.new(
-                message: "failed to fetch passage app",
-                status_code: e.response[:status],
-                body: e.response[:body]
-              )
-      end
+    def get_cache(key)
+      @app_cache[key]
+    end
+
+    def set_cache(key, value)
+      @app_cache[key] = value
     end
```

Comment:
I, will add private. This was a linting error, I forget the specific rule. I can also disable for just this line if that is preferred. I don't feel strongly either way.

Reply:
it prefers [access modifier groups](https://docs.rubocop.org/rubocop/cops_style.html#enforcedstyle_-group-_default_-styleaccessmodifierdeclarations) instead of inline modifiers